WorkflowFunctionBundle/Tests/Functional/Repository/WorkflowFunctionRepositoryTest.php

```diff
@@ -2,6 +2,8 @@
 
 namespace OpenOrchestra\WorkflowFunctionBundle\Tests\Functional\Repository;
 
+use OpenOrchestra\ModelInterface\Repository\Configuration\FinderConfiguration;
+use OpenOrchestra\ModelInterface\Repository\Configuration\PaginateFinderConfiguration;
```

Comment:
There's a problem with implicit chained dependancies: what if cms-bundle is for one reason or another no more dependant on ModerlInterface. WorkflowFunctionBundle collapses